webapi/src/test/scala/org/knora/webapi/http/middleware/AuthenticationMiddlewareZSpec.scala

```diff
@@ -0,0 +1,82 @@
+package org.knora.webapi.http.middleware
```

Comment:
please also include the copyright header

Reply:
done